design-guidelines/interaction-and-feedback.md

```diff
@@ -0,0 +1,59 @@
+# Interaction and Feedback
+
+We can make our apps more intuitive/even easier to use by communicating the status of an interactive element, and providing.
```

Comment:
Is there something missing at the end here?

Reply:
Yea, it should say "providing feedback" 🙈